### PR TITLE
Update License file

### PR DESCRIPTION
**Pull request by Amrit131**
# In this PR/Branch we are going to test my own code from admission trials project.

### We are going to check it for these things -:
- Semantic tags
- Are Html&Css best practices applied
- Is repository up to the github flow.

### CODE REVIEW

**Comment by Amrit131** [comment]
# There are several issues in the pull request.
### Some issues that should be resolved are-:

- Try to solve the Stylelint Linter errors. Linter errors are to make sure you don't have any typos and your code is clean and ready to go into your main codebase.
- You should update readme file in a professional way so that it looks better and gives a description about your project.
- Try to write descriptive Pull requests descriptions. Descriptive pull requests are necessary to let everyone know what updates have you made in a specific PR/branch.
- I think there is room for improvement for styling of the project. The links can be separated and styled in a better way.

**Comment by Amrit131** [comment]
# There are some Html and Css related issues
### While you have used semantic tags for your html there are still issues that can be resolved-:

 - Use html and css best practices for making your project. If you want to know more about that click [here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#html--css-best-practices) Using best practices for html and css shows your ability to write clean code and keeps the codebase neat without errors. 

#### After you have updated the required changes please go ahead and request for a new review from your Student Dashboard.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Change* :recycle: 

Hi @Amrit131, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  Code Review Performed. :heavy_check_mark:




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [x] Good job so far @Amrit131. You've done great making a code review on your previous project. However, there're few things you still need to work on before you can move on to the next project.
     -  You're advised to make use of inline comment provided in github features. Kindly check it [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and make use of it in your review.
- [x] Also, try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where where more information could be got for  more clarification.
- [OPTIONAL] The use of  emojis and/or gifs in the review can also give your review a spice. They make the code review more human-friendly. Good job once again. :handshake:

**Comment by Amrit131** [comment]
@Qoosim I cannot give inline comments because it is saying I am the author of the pull request.
Screenshot (27)
What should I do ?

File: css/style.css [hunk 1]
```diff
@@ -3,3 +3,4 @@
     #main-doc {margin-left: 216px;}
   }
```
Comment (Amrit131) [inline css/style.css:5]: ## I think you should remove this empty lines it will make your code look better and might solve a linter error.

**Comment by Amrit131** [comment]
I have made the required changes.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Amrit131, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

a little more


 ## Good Highlights✔

- You have suggested changes to improve the code
- Some suggested changes have been implemented 


 ## Required Changes ♻️

- As suggested by the previous review, kindly utilize the [GitHub inline comments](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) feature to reference parts of the code that your comment is based on. This will help the code author understand the context of your comment much better.

_Check the comments under the review._

## Optional suggestions

- Following the previous reviewer's suggestion, you could use a couple of emojis and/or gifs in the review make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (Amrit131) [reply css/style.css:5]: @emmiiorji I already did the required changes in this comment.

File: index.html [hunk 2]
```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
+    <!---->
```
Comment (Amrit131) [inline index.html:10]: @emmiiorji I already did the required changes in this comment.

**Comment by Amrit131** [comment]
@emmiiorji I have implemented the required suggestions in comment 3 and 4

**Comment by emmiiorji** [comment]
> @emmiiorji I have implemented the required suggestions in comment 3 and 4

Hi @Amrit131 

Thanks for reaching out for clarification. The previous reviewer had asked that you utilize GitHub inline comments.

Here's an example of a use of GitHub inline comment:
image

I hope this helps.

File: index.html (hunk 2 above)
Comment (Amrit131) [inline index.html:10]: Kindly remove commented line of code. Leaving commented code in your project wouldn't make your code look neat and professional. 🙏

**Review by Amrit131 — COMMENTED** [review]
Kindly check inline comments I made and implement those changes.🙏

File: css/style.css (hunk 1 above)
Comment (Amrit131) [inline css/style.css:5]: Kindly remove this empty line. Removing that empty line will make your code look better and might solve a linter error. 🙏

**Review by Amrit131 — COMMENTED** [review]
Kindly check inline comments I made and implement those changes 🙏

**Review by ichala — APPROVED** [review]
Hi @Amrit131 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Good use of GitHub review features✔️
 - Good Comments✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

- N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**